var/spack/repos/builtin/packages/coreutils/package.py

```diff
@@ -37,5 +39,37 @@ class Coreutils(AutotoolsPackage):
     version('8.29', '960cfe75a42c9907c71439f8eb436303')
     version('8.26', 'd5aa2072f662d4118b9f4c63b94601a6')
     version('8.23', 'abed135279f87ad6762ce57ff6d89c41')
+    version('8.22', '8fb0ae2267aa6e728958adc38f8163a2')
 
     build_directory = 'spack-build'
+
+    variant('mutil', default=False, description='Enable - Multi-threaded cp and md5sum based on GNU coreutils')
+    depends_on('gnutls', when='+mutil')
+    depends_on('libgcrypt', when='+mutil')
+    depends_on('libgpg-error', when='+mutil')
+    depends_on('openmpi', when='+mutil')
+
+    resource(
+       name='mutil',
+       git='https://github.com/pkolano/mutil.git',
+       commit='a60175eba0f02b4674a7dadeaf168d035aace851',
+    )
+
+    @run_before('configure')
+    def enable_mutil(self):
+        if self.spec.satisfies('+mutil'):
+            patch = which('patch')
+            patch("-p1", "--input=mutil/patch/coreutils-8.22.patch")
+
+    @run_after('install')
+    def reinstall_mutil(self):
+        # if the mutil patchis enabled everythng except cp and md5sum is identical to the usual coreutils
+        if self.spec.satisfies('+mutil'):
+            shutil.rmtree(self.prefix.bin)
+            shutil.rmtree(self.prefix.share)
+            mkdirp(self.prefix.bin)
+            mkdirp('{}/man/man1'.format(self.prefix.share))
```

Comment:
Replace `{}` with `{0}` for Python 2.6 support. Also, this might look better as:
```python
self.prefix.share.man.man1
...
self.prefix.man.man1.join('mcp.1')
```